### PR TITLE
Update stonesense.rst

### PR DESCRIPTION
**Pull request by realSquidCoder**
Update the docs to reflect the changes made in [Stonesense PR #120](https://github.com/DFHack/stonesense/pull/120)

### CODE REVIEW

File: docs/plugins/stonesense.rst [hunk 1]
```diff
@@ -68,8 +68,9 @@ at least a dual core CPU to avoid slowing down your game of DF.
 
 Useful links
 ------------
+- `Report issues on Github <https://github.com/DFHack/stonesense/issues>`
```
Comment (myk002) [inline docs/plugins/stonesense.rst:71]:
```suggestion
- Report issues on `Github <https://github.com/DFHack/stonesense/issues>`__
```

File: docs/plugins/stonesense.rst [hunk 2]
```diff
@@ -68,8 +68,9 @@ at least a dual core CPU to avoid slowing down your game of DF.
...
 - :forums:`Official Stonesense thread <106497>` for feedback,
```
Comment (myk002) [inline docs/plugins/stonesense.rst:72]: Keeping a link to the forums is probably still a good idea, but I think the DFHack Discord is the best place for questions, requests, and feedback nowadays. Maybe add a link back to the support page as the first (or second) list item:
```md
- `support`
```

Comment (realSquidCoder) [reply docs/plugins/stonesense.rst:72]: agreed. what is the link ti the discord i should use?

Comment (myk002) [reply docs/plugins/stonesense.rst:72]: use ```
- `support`
```
where there is already information on how to get to the discord

**Review by myk002 — APPROVED** [review]
approved pending merge of https://github.com/DFHack/stonesense/pull/120/files

**Pull request merged**